### DISCUSSION TITLE
Think about wp-contents strategy

### DISCUSSION THREAD

**Discussion by xandfury** [honeynet/honeynet.github.io#9] [General]
All the posts and pages have been converted to markdown, in jekyll format. They currently sit in the development branch. The page export works pretty well. However, we still would need to think about wp-contents, and how we want to accommodate that into the site. 

At first, I wanted to commit the entire directory into the repo. But soon realised that we can probably prune a lot of contents from there. I also faced some git corruption issues like so:

```
[development a335a2d] remove wp-content
error: inflate: data stream error (invalid stored block lengths)
fatal: packed object e69de29bb2d1d6434b8b29ae775ad8c2e48c5391 (stored in .git/objects/pack/pack-9a11e7fc3c38c0534b748b57ad56c2671e0514d7.pack) is corrupt
```  

We do need the essential bits on the site. Rest we can probably keep into to a separate repo (and add references) - pretty much like forensic challenges. 🙂

**Comment by dpisano** [comment]
Take a look at https://wordpress.org/plugins/jekyll-exporter/. It looks like it will take care of wp-contents when it does the export.

**Comment by xandfury** [comment]
Did that already 🙂 I have the wp-contents with me; just don't think it should be part of the repo.

**Comment by dpisano** [comment]
Gotcha. I can think of some options, but they would have some tradeoffs with usability.

**Comment by dpisano** [comment]
For now just put it in the repo. That way we have it in a central location. If needed we can remove it at a letter date.

**Comment by hgascon** [comment]
@xandfury did you put the wp-contents in the development branch?

**Comment by xandfury** [comment]
@hgascon Yep. I added a comment with the update here:  https://github.com/honeynet/docs/issues/1#issuecomment-1483507083

**Comment by xandfury** [comment]
Maybe we should make a list of stuff we really need. I bet there is a lot of data on the site which can be nuked since it's no longer relevant. For e.g. I found a bunch of drafts which really say - look at this awesome link.

**Comment by glaslos** [comment]
Did you consider using [LFS](https://docs.github.com/en/repositories/working-with-files/managing-large-files) for the wp-contents?

**Comment by xandfury** [comment]
@glaslos Yep. The data for the [forensic challenges](https://github.com/honeynet/forensic_challenges) are already on LFS 😄 The main issue is what do we want to keep and what should we purge. I don't see a point in keeping data on LFS if we don't need it.
